Jarvis-Platform/jarvis-studio
# fix(SignIn): Update background-image path

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Fix Sign In page background image path on build.


## Todos
- [x] Import img using TypeScript
- [x] Declare TypeScript module for jpg assets


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout hotfix/sign-in-hero-img-on-build
```

1. Go to `/sign-in` in local => Image should be ok.
2. Build app.
3. Serve `dist/` folder to see the image on a build of the app.


## Impacted Areas in Application
- `src/views/auth/SignIn.vue`
